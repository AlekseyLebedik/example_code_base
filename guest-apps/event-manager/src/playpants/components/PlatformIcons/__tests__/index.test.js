import React from 'react';
import { shallow } from 'enzyme';

import { PlatformIconsBase } from '../index';

describe('PlatformIcons', () => {
  const props = {
    onClick: jest.fn(),
    titles: [
      { id: 1, name: 'MW-PS3', platform: 'PS3' },
      { id: 2, name: 'XB1-PS3', platform: 'XB1' },
      { id: 3, name: 'XB1-Vanilla (Valor)', platform: 'XB1' },
    ],
    projectColors: [{}],
  };

  const root = shallow(<PlatformIconsBase {...props} />);
  it('renders default props', () => {
    expect(root).toMatchSnapshot();
  });
});
