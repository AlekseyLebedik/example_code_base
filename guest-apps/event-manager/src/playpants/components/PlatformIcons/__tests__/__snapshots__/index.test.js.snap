// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlatformIcons renders default props 1`] = `
Array [
  <WithStyles(ForwardRef(Tooltip))
    TransitionComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "propTypes": Object {
          "children": [Function],
          "disableStrictModeCompat": [Function],
          "in": [Function],
          "onEnter": [Function],
          "onEntered": [Function],
          "onEntering": [Function],
          "onExit": [Function],
          "onExited": [Function],
          "onExiting": [Function],
          "style": [Function],
          "timeout": [Function],
        },
        "render": [Function],
      }
    }
    key="MW-PS3"
    title="MW-PS3"
  >
    <span>
      <PlatformIcon
        className=""
        color="#212121"
        platform="PS3"
      />
    </span>
  </WithStyles(ForwardRef(Tooltip))>,
  <WithStyles(ForwardRef(Tooltip))
    TransitionComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "propTypes": Object {
          "children": [Function],
          "disableStrictModeCompat": [Function],
          "in": [Function],
          "onEnter": [Function],
          "onEntered": [Function],
          "onEntering": [Function],
          "onExit": [Function],
          "onExited": [Function],
          "onExiting": [Function],
          "style": [Function],
          "timeout": [Function],
        },
        "render": [Function],
      }
    }
    key="XB1-PS3"
    title="XB1-PS3"
  >
    <span>
      <PlatformIcon
        className=""
        color="#212121"
        platform="XB1"
      />
    </span>
  </WithStyles(ForwardRef(Tooltip))>,
  <WithStyles(ForwardRef(Tooltip))
    TransitionComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "propTypes": Object {
          "children": [Function],
          "disableStrictModeCompat": [Function],
          "in": [Function],
          "onEnter": [Function],
          "onEntered": [Function],
          "onEntering": [Function],
          "onExit": [Function],
          "onExited": [Function],
          "onExiting": [Function],
          "style": [Function],
          "timeout": [Function],
        },
        "render": [Function],
      }
    }
    key="XB1-Vanilla (Valor)"
    title="XB1-Vanilla (Valor)"
  >
    <span>
      <PlatformIcon
        className=""
        color="#212121"
        platform="XB1"
      />
    </span>
  </WithStyles(ForwardRef(Tooltip))>,
]
`;
